contracts/nft-querier/src/state.rs

```diff
@@ -0,0 +1,9 @@
+use cosmwasm_schema::cw_serde;
+use cw_storage_plus::Item;
+
+#[cw_serde]
+pub struct Config {
+    pub factory_contract: String,
```

Comment:
Always store addresses as `cosmwasm_std::Addr` instead of `String`. This way you will be forced to validate an address (using `deps.api.addr_validate()`) before saving it.